concepts/Common/Distribution Channels/05 CDN/05 CDN.md

```diff
@@ -0,0 +1,60 @@
+## Scripts
```

Comment:
This article looks kinda weird. Just a list of all files available via CDN will follow (see [the example](https://js.devexpress.com/Documentation/Guide/Getting_Started/Installation/CDN_Services/#Subresource_Integrity_Hashes) of Subresource Integrity Hashes). We had this part of the article to explain which files are needed in different cases, now we don't have this information. I think we should keep at least the first paragraph from the previous article.